mailet/standard/src/main/java/org/apache/james/transport/mailets/ClamAVScan.java

```diff
@@ -738,16 +737,16 @@ protected void ping(InetAddress address) throws Exception {
         int ping = 1;
         for (; ; ) {
             if (isDebug()) {
-                LOGGER.debug("Trial #" + ping + "/" + getMaxPings() + " - creating socket connected to " + address + " on port " + getPort());
+                LOGGER.debug("Trial #{}/{} - creating socket connected to {} on port {}", ping, getMaxPings(), address, getPort());
             }
             try {
                 socket = new Socket(address, getPort());
                 break;
             } catch (ConnectException ce) {
-                LOGGER.debug("Trial #" + ping + "/" + getMaxPings() + " - exception caught: " + ce.toString() + " while creating socket connected to " + address + " on port " + getPort());
+                LOGGER.debug("Trial #{}/{} - exception caught: {} while creating socket connected to {} on port {}", ping, getMaxPings(), ce, address, getPort());
```

Comment:
This log line seems still poor.

Something like 

```
LOGGER.debug("Trial #{}/{} - exception caught while creating socket connected to {} on port {}", ping, getMaxPings(), address, getPort(), ce);
```

That would avoid loosing the stacktrace...

Reply:
Done